docs/source/deploying/production.md

```diff
@@ -0,0 +1,182 @@
+---
+
+# Production deployment
+
+```{versionadded} Volto 18.0.0-alpha.43
+```
+
+```{deprecated} 18.0.0-alpha.43
+The Volto app approach is deprecated. For production deployments, use [Cookieplone](https://github.com/plone/cookieplone) with the `sub/frontend_project` template instead.
+```
+
+## Overview
+
+For production deployments of Volto, the recommended approach is to use [Cookieplone](https://github.com/plone/cookieplone) with the `sub/frontend_project` template. This creates a production-ready project structure that uses Volto as a library rather than as a standalone app.
+
+This approach provides:
+- A clean, minimal boilerplate project structure
+- Better separation of concerns
+- Easier maintenance and upgrades
+- Production-ready deployment configuration
+
+## Creating a production project
+
+### Using Cookieplone with the frontend_project template
+
+To create a production-ready Volto project, use Cookieplone's `sub/frontend_project` template:
+
+```shell
+pipx run cookieplone sub/frontend_project
+```
+
+```{note}
+The `sub/frontend_project` template creates a minimal boilerplate project structure for production deployments. This is different from the `frontend_addon` template, which is used for creating Volto add-ons. For full-stack Plone projects (with both frontend and backend), use the `project` template instead.
+```
+
+This command will:
+1. Prompt you for project details (project name, description, etc.)
+2. Generate a boilerplate project structure
+3. Set up a `package.json` that depends on Volto
+4. Include deployment scripts and configuration
+
+The generated project will have a minimal structure, with most functionality coming from Volto as a dependency. The project essentially becomes a thin wrapper around Volto, defining your specific configuration and customizations.
+
+### Project structure
+
+After generation, your project will have a structure similar to:
+
+```
+my-volto-project/
+├── package.json          # Defines Volto as a dependency
+├── src/
+│   └── config.js         # Your Volto configuration
+├── public/               # Static assets
+└── build/                # Production build output (after build)
+```
+
+## Building for production
+
+Once you have your project structure, build it for production:
+
+```bash
+pnpm install
+pnpm build
+```
+
+For production builds with specific environment variables:
+
+```bash
+PORT=3000 RAZZLE_API_PATH=https://mywebsite.com/api pnpm build
+```
+
+The build process will:
+- Compile your React application
+- Bundle all dependencies
+- Create optimized production assets in the `build` directory
+
+## Deployment options
+
+### Using a Node.js base image
+
+If you need to use your own Node.js base image (rather than the official `plone-frontend` images), you can:
+
+1. **Build the project** in your CI/CD pipeline or locally
+2. **Copy the built project** to your production image
+3. **Start the production server** using the built files
```

Comment:
Please don't add strong formatting where it's not helpful.

```suggestion
1.  Build the project in your CI/CD pipeline or locally.

    ```shell
    pnpm install
    PORT=3000 RAZZLE_API_PATH=https://mywebsite.com/api pnpm build
    ```

2.  Copy the built project from the {file}`./build` directory to your production image.
3.  Start the production server using the built files.

    ```shell
    NODE_ENV=production node build/server.js
    ```
    
```